api/routes/globalRoutes.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = function(app) {
+
+    // Use Global View controller to pull data from the model (MySQL DB), 
+    // convert it to JSON and return it to the route.
+
+    var patients = require('../controllers/globalController');
```

Comment:
better to use global instead of patients

Reply:
Done